_layouts/default.html

```diff
@@ -95,20 +97,20 @@
          </div>
          <div class="col-sm-6">
            <div class="nav justify-content-end">
-             <a class="nav-link text-black-50" href="#">
-               <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/fb.svg#fb"></use></svg>
+             <a class="nav-link text-black-50" href="#" aria-label="MapLibre on Facebook">
+               <svg aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/fb.svg#fb"></use></svg>
              </a>
-             <a class="nav-link text-black-50" href="#">
-               <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/linkedin.svg#linkedin"></use></svg>
+             <a class="nav-link text-black-50" href="#" aria-label="MapLibre on LinkedIn">
+               <svg aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/linkedin.svg#linkedin"></use></svg>
              </a>
-             <a class="nav-link text-black-50" href="#">
-               <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/twt.svg#twt"></use></svg>
+             <a class="nav-link text-black-50" href="#" aria-label="MapLibre on Twitter">
+               <svg aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/twt.svg#twt"></use></svg>
              </a>
-             <a class="nav-link text-black-50" href="#">
-               <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/ytb.svg#ytb"></use></svg>
+             <a class="nav-link text-black-50" href="#" aria-label="MapLibre on YouTube">
+               <svg aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/ytb.svg#ytb"></use></svg>
              </a>
-             <a class="nav-link text-black-50" href="#">
-               <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/ig.svg#ig"></use></svg>
+             <a class="nav-link text-black-50" href="#" aria-label="MapLibre on Instagram">
+               <svg aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><use href="{{ "/" | relative_url }}img/icon/ig.svg#ig"></use></svg>
```

Comment:
These links are missing link destinations which is not ideal, but I've pre-populated aria-labels in case they're to be used in the future.